### DISCUSSION TITLE
LESSON 8

### DISCUSSION THREAD

**Discussion by mrthedude** [Cyfrin/foundry-full-course-cu#529] [General]
I downloaded the Live Server Extension with no issues but when I try to  open ```index.html``` with it, nothing happens other than I see the default ```Port: 5500``` at the bottom but with a little blocked symbol before it. I've attached a screenshot of the bottom bar before starting Live Server and then after for reference. Here's the troubleshooting I've done so far:

Uninstalled then reinstalled the extension-- no change

Restarted VS Code with ```Developer: Reload Window``` in the command bar up top-- no change

Changed the extension settings to randomize the port selected with ```"liveServer.settings.port": 0``` to see if it was an issue with that particular port-- no change

Changed the extension settings to explicitly have chrome be the default browser with ```"liveServer.settings.CustomBrowser": "chrome"``` (already set in my computer settings to default to chrome btw)-- no change 

Paused my firewall-- no change 

Disabled all my browser extensions (chrome)-- no change 

For what it's worth, I have a feeling that it is an issue with my security software but I don't know why pausing the firewall didn't solve the issue if that's the case. I know this is a minor issue and mostly irrelevant to the overall course but I am curious to try and find out what could be the issue, and I might be looking more into HTML so this extension would be great to use if it worked.

Thankyou in advance
![Screenshot 2023-08-03 180423](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/a7ed2419-9777-4e16-ab89-f709c2af0516)

![Screenshot 2023-08-03 180402](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/9cd1e337-1885-4d98-9fa1-14e2fc75d515)

**Comment by shikhar229169** [comment]
If you are only concerned about this '🚫' symbol, it represents to close the live server.

**Reply by mrthedude** [reply]
oh so then any ideas why nothing pops up after I click the 'Go Live' button at the bottom? Isn't a browser window supposed to open up?

**Reply by shikhar229169** [reply]
Yeah, I also wonder why it doesn't pop up, so what I did was manually searched the link in browser 
```
http://127.0.0.1:5500/
```
And it worked.

**Reply by mrthedude** [reply]
It works when I manually searched it in my browser as well, weird that it's not automatic. I even added the Live Server browser extension to see if that'd help anything but no luck. Thankyou for taking the time to help!

**Comment by theirrationalone** [comment]
Try to do this...

In settings.json file...

```json
"liveServer.settings.NoBrowser": false
```

If that doesn't work, then try to reach manually. (Open Browser and search "localhost:port")...
i.e.,
```link
http://127.0.0.1:5500
```

**Reply by mrthedude** [reply]
I changed the settings but no luck, still doesn't automatically pop up so I searched it manually. My hunch is that it's my firewall by in any case I'll just manually do it. Thankyou for taking the time to help!

**Reply by shikhar229169** [reply]
The thing is that live server pops up the browser when we are are not in wsl. So, the problem is actually related to wsl.

**Reply by shikhar229169** [reply]
I asked chat gpt about it, but the solution provided by it doesn't work.

**Reply by theirrationalone** [reply]
> I asked chat gpt about it, but the solution provided by it doesn't work.

Ohh yeassss, you may right here, Hey @mrthedude please try to install the same extension for linux wsl env. This might set permissions for wsl to interact with brave/chrome/etc.

Open extensions menu, and do install that extension in wsl linux menu.

**Reply by mrthedude** [reply]
@theirrationalone i think that may be the reason yeah, now i just have to figure out how to do that 🤣 . You're talking about in my regular terminal correct? (attached a screenshot)
![Screenshot 2023-08-06 193900](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/cdb4fdf8-e08f-43bb-a731-b1e0216bcb77)

**Reply by theirrationalone** [reply]
> @theirrationalone i think that may be the reason yeah, now i just have to figure out how to do that 🤣 . You're talking about in my regular terminal correct? (attached a screenshot)
> ![Screenshot 2023-08-06 193900](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/cdb4fdf8-e08f-43bb-a731-b1e0216bcb77)
> 

Naah, when you're on vscode and wsl is active, then in extensions menu you have choice in which dev. env. You wanna install extensions. 

But you are checking right although using powershell. But you wouldn't may have access to linux instance there.

**Reply by mrthedude** [reply]
oh i see what you're saying... i ended up fixing the issue by downloading an updated live server extension
![Screenshot 2023-08-07 163536](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/b652a160-c3f4-4091-b94f-8d952f2884ac)

**Reply by theirrationalone** [reply]
Solved.... Wowww, but i meant something similar, leave everything now it's working that's Great.

**Reply by mrthedude** [reply]
hahahaha thanks for all your help!

**Comment by mrthedude** [comment]
> I asked chat gpt about it, but the solution provided by it doesn't work.
i was asking chatGPT too lol but nothing useful
